server/game/cards/characters/04/myrcellabaratheon.js

```diff
@@ -0,0 +1,36 @@
+const _ = require('underscore');
+const DrawCard = require('../../../drawcard.js');
+
+class MyrcellaBaratheon extends DrawCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            condition: () => (
+                !this.getKingsInPlay() &&
+                this.game.currentChallenge &&
+                this.game.currentChallenge.challengeType === 'power'),
+            match: this,
+            effect: ability.effects.doesNotKneelAsDefender()
+        });
+
+        this.persistentEffect({
+            condition: () => !this.getKingsInPlay(),
+            match: this,
+            effect: ability.effects.addKeyword('Renown')
+        });
+    }
+
+    getKingsInPlay() {
```

Comment:
Ugh, sorry about that mess. I'll take a good look at it later so I can make sure I understand what went wrong. Thanks as usual.